docs/components/modeler/desktop-modeler/troubleshooting.md

```diff
@@ -12,24 +15,39 @@ You can start Desktop Modeler with the [`--disable-plugins` flag](./flags/flags.
 
 Depending on your operating system, you can find Desktop Modeler logs in different places:
 
-### Windows
+<Tabs defaultValue="windows" values={
```

Comment:
:+1:

Reply:
@pepopowitz Do you have thoughts here? Can always create a separate issue for a closer look.

Reply:
https://docusaurus.io/docs/markdown-features/tabs#query-string

i think you are looking for a `queryString` here.

Reply:
Resolved in latest commit 👍